rfcs/text/0000-dss-deletion-daemon.md

```diff
@@ -0,0 +1,416 @@
+ 
+### Tombstones
+
+Tombstones are markers places in the DSS to indicate previously existing data has been removed. Tombstones exist for
+files and bundles stored in the DSS. Tombstones come in two varieties, versioned and unversioned tombstones. A 
+version tombstone marks the deletion of a specific version of a file or bundle. This prevents that same version of data 
+from being reuploaded or retrieved. An unversioned tombstone does not specify a version and prevent a file or bundle 
+using the same UUID from being reuploaded or retrieved. Whether a tombstone is version or unversioned is determined
+during a deletion request. If the version parameter is present in the request, then a versioned tombstone is created. If
+no version is specified then an unversioned tombstone is created that is applied to all the bundles with that UUID. A 
+versioned tombstone is created for all bundles and files that are logically or physically deleted as a side affect of a 
+deletion operations. Unversioned tombstones should be used with caution for they effectively retired an existing UUID 
+prevent users from uploaded a corrected version of the data in the future.
+
+### Primary indexes
+
+Bundle tombstones are indexed verbatim by the Data Store. The JSON schema
+for deletion markers is defined such that there are no conflicts with the
+top-level keys in the index documents for regular bundles (`files`, `manifest`,
+`uuid`, `version` etc). Bundle tombstones are therefore compatible with
+regular bundle documents with respect to in the same index even if
+Elasticsearch's dynamic mapping is enabled.
+
+### Replication
+
+When a bundle or file is deleted it is removed from all replicas.
+
+### Consistency
+
+Race conditions with Elasticsearch can arise when deleting files or bundles. The confirmation code is used to ensure
+that the state of the index has not changes between deletion requests. If the state has changed another request to delete
+should be made and the effected files and bundles should be verified before sending the confirmation to delete.
+
+### Secondary indexes
+
+The Data Store can optionally notify subscribers about the creation of a bundle
+tombstone. Secondary index services need to register a subscription using
+a query that matches deletion markers. When they are notified about a deletion
+marker, they need to take the necessary actions to remove the data from their
+database or index.
+
+### Derived data
+
+The metadata in bundles containing derived data must reference the input bundle
+versions. Administrators are responsible for tracking down data bundles with
+data derived from data in deleted bundles and to issue deletion requests for
+them.
+
+### Tiered storage
+
+Physical deletion of blobs is effective across all tiered storage classes in
+the underlying blob store. Additional cost may be incurred for deleting data recently moved to a
+slower access storage tier. There is no additional delays cause when deleting from a 
```

Comment:
There is no additional delays cause -> ?

Reply:
changed to "There is no additional time delay caused when deleting from a tiered storage system such as AWS glacier."